src/DaedalusCompiler/Compilation/Compiler.cs

```diff
@@ -52,14 +57,21 @@ public string GetBuiltinsPath()
                 for (int i = 0; i < paths.Length; i++)
                 {
                     if (verbose) Console.WriteLine($"[{i + 1}/{paths.Length}]Compiling: {paths[i]}");
-
-                    // create parser for specific file
-                    var parser = GetParser(paths[i]);
-
+                    DaedalusParser parser = GetParser(paths[i]);
```

Comment:
Currently we load from the doubly file with daedalus script, once in GetParser method ( with antl4 AntlrFileStream ) and second one in OuBuilder ParseFile, we should load file once and pass string or byte[] to GetParser and to OuBuilder.ParseFile. Instead of AntlrFileStream we can use AntlrInputStream which can use byte stream